app/views/AnalyticalFramework/UserTable/AddUserModal/index.tsx

```diff
@@ -120,6 +137,57 @@ function AddUserModal(props: Props) {
 
     const error = getErrorObject(riskyError);
 
+    const [
+        addEditFrameworkMembership,
+        {
+            loading: addEditFrameworkMembershipLoading,
+        },
+    ] = useMutation<AnalysisMembershipAddEditMutation, AnalysisMembershipAddEditMutationVariables>(
+        ANALYSIS_MEMBERSHIP_ADD_EDIT,
+        {
+            onCompleted: (response) => {
+                if (!response.analysisFramework?.analysisFrameworkMembershipBulk?.result) {
+                    return;
+                }
+
+                const {
+                    result,
+                } = response.analysisFramework.analysisFrameworkMembershipBulk;
+
+                const ok = isDefined(result) && result?.length > 0;
+
+                if (ok) {
+                    if (userValue?.id) {
+                        alert.show(
+                            'Successfully updated user to the analytical framework.',
+                            { variant: 'success' },
+                        );
+                    } else {
+                        alert.show(
+                            'Successfully added user to the analytical framework.',
+                            { variant: 'success' },
+                        );
+                    }
```

Comment:
Can we not fetch ok directly from the server?

Reply:
there is no ok response in bulk mutation